cves/CVE-2002-0392.yml

```diff
@@ -154,37 +169,44 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
+        There is a basic level of input distrust in the original code (VCC) but it
+        doesn't go so far as to validate the input against wacky values that may
```

Comment:
As much as I love the phrase "wacky values," may want to rewrite this to sound a bit more professional.

Reply:
Yep, good call.